snoop/data/analyzers/image_classification.py

```diff
@@ -0,0 +1,147 @@
+"""Task to call a service that runs object detection and/or image classification on images"""
+
+import requests
+from .. import models
+from django.conf import settings
+from ..tasks import SnoopTaskBroken, snoop_task, returns_json_blob
+import json
+import io
+from PIL import Image, UnidentifiedImageError
+
+
+PROBABILITY_LIMIT = 20
+
+"""Based on https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#image-file-formats"""
```

Comment:
docstrings for assignments should go after it, check the docs output at https://hoover-snoop2--444.org.readthedocs.build/en/444/reference/snoop/data/analyzers/image_classification/ -- the link goes under `PROBABILITY_LIMIT` which is not what you want